test/sparse/sparse.jl

```diff
@@ -646,321 +655,318 @@ end
+    Base.SparseArrays.dropstored!(A, :, 2:4)
+    @test nnz(A) == 0
+    # --> Introduce nonzeros in every other row
+    A[1:2:9, :] = 1
+    @test nnz(A) == 50
+    # --> Test dropping a block of the matrix towards the upper left
+    Base.SparseArrays.dropstored!(A, 2:5, 2:5)
+    @test nnz(A) == 42
+end
+
+@testset "issue #7507" begin
     @test (i7507=sparsevec(Dict{Int64, Float64}(), 10))==spzeros(10)
 end
 
-@testset "Issue 7650" begin
-    let S = spzeros(3, 3)
-        @test size(reshape(S, 9, 1)) == (9,1)
-    end
-
-    let X = eye(5), M = rand(5,4), C = spzeros(3,3)
-        SX = sparse(X); SM = sparse(M)
-        VX = vec(X); VSX = vec(SX)
-        VM = vec(M); VSM1 = vec(SM); VSM2 = sparsevec(M)
-        VC = vec(C)
-        @test VX == VSX
-        @test VM == VSM1
-        @test VM == VSM2
-        @test size(VC) == (9,)
-        @test nnz(VC) == 0
-        @test nnz(VSX) == 5
-    end
-end
-
-@testset "Issue 7677" begin
-    let A = sprand(5,5,0.5,(n)->rand(Float64,n)), ACPY = copy(A)
-        B = reshape(A,25,1)
-        @test A == ACPY
-        C = reinterpret(Int64, A, (25, 1))
-        @test A == ACPY
-        D = reinterpret(Int64, copy(B))
-        @test C == D
-    end
-end
-
-@testset "Issue #8225" begin
+@testset "issue #7650" begin
+    S = spzeros(3, 3)
+    @test size(reshape(S, 9, 1)) == (9,1)
+
+    X = eye(5)
+    M = rand(5,4)
+    C = spzeros(3,3)
+    SX = sparse(X); SM = sparse(M)
+    VX = vec(X); VSX = vec(SX)
+    VM = vec(M); VSM1 = vec(SM); VSM2 = sparsevec(M)
+    VC = vec(C)
+    @test VX == VSX
+    @test VM == VSM1
+    @test VM == VSM2
+    @test size(VC) == (9,)
+    @test nnz(VC) == 0
+    @test nnz(VSX) == 5
+end
+
+@testset "issue #7677" begin
+    A = sprand(5,5,0.5,(n)->rand(Float64,n))
+    ACPY = copy(A)
+    B = reshape(A,25,1)
+    @test A == ACPY
+    C = reinterpret(Int64, A, (25, 1))
+    @test A == ACPY
+    D = reinterpret(Int64, copy(B))
+    @test C == D
+end
+
+@testset "issue #8225" begin
     @test_throws ArgumentError sparse([0],[-1],[1.0],2,2)
 end
 
-@testset "Issue #8363" begin
+@testset "issue #8363" begin
     @test_throws ArgumentError sparsevec(Dict(-1=>1,1=>2))
 end
 
-@testset "Issue #8976" begin
+@testset "issue #8976" begin
     @test conj.(sparse([1im])) == sparse(conj([1im]))
     @test conj!(sparse([1im])) == sparse(conj!([1im]))
 end
 
-@testset "Issue #9525" begin
+@testset "issue #9525" begin
     @test_throws ArgumentError sparse([3], [5], 1.0, 3, 3)
 end
 
 @testset "indmax, indmin, findmax, findmin" begin
-    let S = sprand(100,80, 0.5), A = Array(S)
-        @test indmax(S) == indmax(A)
-        @test indmin(S) == indmin(A)
-        @test findmin(S) == findmin(A)
-        @test findmax(S) == findmax(A)
-        for region in [(1,), (2,), (1,2)], m in [findmax, findmin]
-            @test m(S, region) == m(A, region)
-        end
-    end
-
-    let S = spzeros(10,8), A = Array(S)
```

Comment:
Absent input from other parties, let's not worry about it.

Having another pair of eyes skim this diff would be worthwhile. Not certain whether that's likely to happen given the holidays and upcoming release though. So absent objections or requests for time, I plan to merge this Saturday morning PST.

Thanks again for seeing this through @lbollar!